src/error-boundary/FullScreenErrorView.tsx

```diff
@@ -58,25 +54,36 @@ export function FullScreenErrorView({onRestartApp, errorCode}: ErrorProps) {
 const useStyles = StyleSheet.createThemeHook((theme) => ({
   safearea: {
     flex: 1,
+    justifyContent: 'space-between',
     backgroundColor: theme.color.background.neutral[2].background,
   },
+  svg: {
+    marginTop: 200,
```

Comment:
It looks like there is a lot of spacing; maybe centring is better? Or does it look on small screens?

Reply:
removed the margin, it looks ok